pkg/bot.go

```diff
@@ -464,9 +475,14 @@ func (b *bot) hasPreviewSiteCapability(email, org string) (bool, error) {
 }
 
 func (b *bot) previewSite(args ResponseRequest) string {
+	if args.Email == "" {
```

Comment:
moving this higher up the call to save CPU some cycles, when user has no public email, we can stop there and let them know.